app/services/i_scream_service.rb

```diff
@@ -1,7 +1,8 @@
 class IScreamService 
 
   def conn 
-    Faraday.new('http://localhost:9393')
+    # Faraday.new('http://localhost:9393')
+    Faraday.new('https://i-scream-microservice.herokuapp.com')
```

Comment:
If you want to run local tests, you have to swap these two lines

Reply:
Good to know. I'm guessing we should switch it back once it's ready so it can pass on Travis